Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM repo.backbase.com/backbase-docker-releases/web-base:2.0.1
+FROM repo.backbase.com/backbase-docker-releases/web-base:latest
```

Comment:
We have a requirement from the security team not to use the latest tag because it is dynamic and leads to uncontrollable updates for the app.